### PR TITLE
Remove deprecation warning, specify 3.x compatability

### PR DESCRIPTION
**Pull request by awmuncy**
Removes deprecation warning, instead specifying that JETI is only compatible with the 3.x version of @tableland/sdk.

- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +7,10 @@ This repository is no longer actively maintained and is deprecated.
 
 A TypeScript/JavaScript library for inserting and extracting IPFS data to and from Tableland queries using the `@tableland/sdk`.
 
+This library is only compartible with version 3 of `@tableland/sdk`.
```
Comment (joewagner) [inline README.md:10]: Looks like a spelling error, should it be "compatible"?

**Review by joewagner — COMMENTED** [review]
I see one spelling issue, also looks like the tests aren't passing but we will have to do a different pr to fix that.

File: README.md (hunk 1 above)
Comment (awmuncy) [reply README.md]: Fixed.

**Review by joewagner — APPROVED** [review]
Looks good!

**Comment by awmuncy** [comment]
> I see one spelling issue, also looks like the tests aren't passing but we will have to do a different pr to fix that.

Yep. The testing is a major issue. To bring JETI to compatibility with v4, and our current standards for libraries we're actively maintaining, we'd need to get a good test suite passing, which at the moment it does not. I wanted to get a simple note out there so folks didn't show up to the library and assume it's compatible with our current work.

**Pull request merged**